hmri_create_MTProt.m

```diff
@@ -1175,6 +1177,14 @@ function PDcalculation(fA, fTPM, mpm_params)
 end 
 mpm_params.ncon = ccon; % number of contrasts available
 
+% Throw a warning if the contrasts PDw, MTw and T1w have non-matching
+% dimensions
+raw = [jobsubj.raw_mpm.MT; jobsubj.raw_mpm.PD; jobsubj.raw_mpm.T1];
+sts = spm_check_orientations(spm_vol(char(raw)));
+if ~sts
```

Comment:
I think it might be cleaner to avoid defining a variable here. Especially since it's anyway not clear to me what `sts` stands for.
```suggestion
if ~spm_check_orientations(spm_vol(char(raw)))
```

Reply:
Ah, looking at it again, perhaps we should instead use both output arguments of `spm_check_orientations`, so that the information about which files don't match gets saved to the log, rather than just printed to the console.

Reply:
Hmm... evidently we would need to use [`evalc`](https://mathworks.com/help/matlab/ref/evalc.html) to do this, as the verbose output only goes to the console.

Reply:
I think it would be very useful to list the file dimensions in the log, as many people don't look at the console and only start looking at the logs after they notice their images look bad after all the processing is done. 

You're right that we could adapt `spm_check_orientations` (calling it `hmri_check_orientations`?) to add the "verbose" string to the `str` output, though. Perhaps that would be cleaner (and safer) than using `evalc`.

Reply:
But why rewrite the code that's already in `spm_check_orientations`?

Reply:
I think it's a much more common problem than you think it is. Previously it would just crash, and people would get confused. Now it gives useful information to help diagnose the problem.

I'll push a branch that shows that we really don't need to do much to `spm_check_orientations` to make it suitable for us.

Reply:
See here: https://github.com/hMRI-group/hMRI-toolbox/tree/add_hmri_check_orientations

Reply:
Try running the old version of `hmri_check_orientations` without any output arguments. It will just give an error without any of the warning messages.

Reply:
And the problem with non-matching dimensions/orientations happens all the time. The two main reasons are scripts that accidentally include too much data or mistaken acquisitions on the scanner.

I don't think anyone will try to use our function for surface files. If you want to be sure, though, we could add a check at the beginning that the input file is a NIfTI file.

Reply:
(Although we could instead just change the function name to `hmri_check_nifti_orientations`)

Reply:
Shall I make the change? I don't want to change things if you're also working on it.